docs/modules/ROOT/pages/contributor-guide.adoc

```diff
@@ -124,3 +124,134 @@ Review the result visually.
++
+[source,shell]
+----
+$ {
+   sed -i '/integration-test/d' "extensions-jvm/${EXT}/pom.xml"
+   sed -i "/<module>${EXT}<\/module>/d" "extensions-jvm/pom.xml"
+   git mv "extensions-jvm/${EXT}/integration-test/" "integration-tests/${EXT}"
+   git mv "extensions-jvm/${EXT}" "extensions/${EXT}"
+   sed -i -r "s/(.*)activemq(.*)/\1activemq\2\n\1${EXT}\2/g" extensions/pom.xml
+   sed -i -r "s/(.*)activemq(.*)/\1activemq\2\n\1${EXT}\2/g" integration-tests/pom.xml
+   sed -i "s/camel-quarkus-grpc-parent/camel-quarkus-integration-tests/g" "integration-tests/${EXT}/pom.xml"
+   sed -i "s/camel-quarkus-grpc-integration-test/camel-quarkus-integration-test-${EXT}/g" "integration-tests/${EXT}/pom.xml"
+   sed -i -r "s/Quarkus :: (.*) :: Integration Test/Quarkus :: Integration Test :: \1/g" "integration-tests/${EXT}/pom.xml"
+  }
+----
+
+5. Add the native profile at the end of `integration-tests/${EXT}/pom.xml`:
++
+[source,xml]
+----
+    <profiles>
+        <profile>
+            <id>native</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.package.type>native</quarkus.package.type>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+----
+
+6. Remove the warning link:https://quarkus.io/guides/writing-extensions#build-step-processors[build step] from `extensions/${EXT}/deployment/src/main/java/org/apache/camel/quarkus/component/${EXT}/deployment/${EXT}Processor.java`:
++
+[source,java]
+----
+    /**
+     * Remove this once this extension starts supporting the native mode.
+     */
+    @BuildStep(onlyIf = NativeBuild.class)
+    @Record(value = ExecutionTime.RUNTIME_INIT)
+    void warnJvmInNative(JvmOnlyRecorder recorder) {
+        JvmOnlyRecorder.warnJvmInNative(LOG, FEATURE); // warn at build time
+        recorder.warnJvmInNative(FEATURE); // warn at runtime
+    }
+----
+
+7. Create a native test at `integration-tests/${EXT}/src/test/java/org/apache/camel/quarkus/component/${EXT}/it/${EXT}IT.java`
+
+8. Check `extensions/${EXT}/runtime/src/main/resources/META-INF/quarkus-extension.yaml`:
+* Ensure guide is set to `"https://quarkus.io/guides/camel"`
+* Ensure keyword `camel` is present
```

Comment:
I do not think this is necessary. The keywords are not shown anywhere, they just help to find extensions via search on code.quarkus.io https://github.com/quarkusio/code.quarkus.io/blob/21044d84bee2ca82ac3939df00b9d0646a75f614/src/main/frontend/src/code-quarkus/pickers/extensions-picker-helpers.ts#L13 The word `Camel` is already there in the extension name so all our extensions should be findable via `camel` anyway https://github.com/quarkusio/code.quarkus.io/blob/21044d84bee2ca82ac3939df00b9d0646a75f614/src/main/frontend/src/code-quarkus/pickers/extensions-picker-helpers.ts#L12

Reply:
I thought there was a plugin preventing the build but I will double-check.